target_chains/ethereum/contracts/forge-test/Executor.t.sol

```diff
@@ -118,14 +122,68 @@ contract ExecutorTest is Test, WormholeTestUtils {
         testExecute(
             address(callable),
             abi.encodeWithSelector(ICallable.foo.selector),
-            1
+            1,
+            0
         );
         assertEq(callable.fooCount(), c + 1);
         assertEq(callable.lastCaller(), address(executor));
         // Sanity check to make sure the check above is meaningful.
         assert(address(executor) != address(this));
     }
 
+    function testCallWithValueSucceeds() public {
+        callable.reset();
+
+        uint32 c = callable.fooCount();
+        assertEq(callable.lastCaller(), address(bytes20(0)));
+
+        uint value = 1;
+        vm.deal(address(executor), value);
+
+        testExecute(
+            address(callable),
+            abi.encodeWithSelector(ICallable.fooPayable.selector),
+            1,
+            value
+        );
+        assertEq(callable.fooCount(), c + 1);
+        assertEq(callable.lastCaller(), address(executor));
+        // Sanity check to make sure the check above is meaningful.
+        assert(address(executor) != address(this));
```

Comment:
I think this test should also check that the balance of the `callable` contract increased